config/application.rb

```diff
@@ -27,7 +27,7 @@ class Application < Rails::Application
     config.middleware.insert_before 0, "Rack::Cors" do
       allow do
         origins '*'
-        resource '*', :headers => :any, :methods => [:get, :post, :put, :patch, :options]
+        resource '*', headers: :any, methods: [:get, :post, :put, :patch, :delete, :options]
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.